src/esp/gfx/Simulator.h

```diff
@@ -62,6 +62,10 @@ class Simulator {
   void saveFrame(const std::string& filename);
 
   RenderingTarget::ptr createRenderingTarget(int width, int height);
+  inline RenderingTarget::ptr createRenderingTarget(
+      const Magnum::Vector2i& size) {
+    return createRenderingTarget(size[0], size[1]);
+  }
```

Comment:
Why not have just the `Vector2i` overload? On the call site it's just about doing `create_rendering_target([width, height])` if you have those separate (or `{}` in case of C++). Magnum does the same, following Python's philosophy of "provide only one way to do things" there's never separate vector/scalar overloads.

Then I would change `createRenderingTarget()` and `RenderingTarget` constructor to also take `Vector2i` instead of two scalars, so it's consistent and one doesn't need to pack/unpack the components every time.

Btw, `inline` inside class definition is redundant, that's implicit ([spec](https://en.cppreference.com/w/cpp/language/inline), see the second paragraph).